SmartDeviceLink/SDLGetVehicleData.h

```diff
@@ -118,9 +142,44 @@ NS_ASSUME_NONNULL_BEGIN
  *  @param turnSignal                  Get turnSignal data
  *  @param vin                         Get vin data
  *  @param wiperStatus                 Get wiperStatus data
- *  @return                            A SDLGetVehicleData object
+ * @return A SDLGetVehicleData object
```

Comment:
Consistency is good - if you have one space between @return and description , you should change the others; Also there is missing space before @return